### PR TITLE
Updated add_sid_history (misc::addsid is deprecated in mimikatz) Added ComputerName argument

### PR DESCRIPTION
**Pull request by ilanisme**

### CODE REVIEW

File: empire/server/modules/powershell/persistence/misc/add_sid_history.py [hunk 1]
```diff
@@ -28,14 +28,13 @@ def generate(main_menu, module: PydanticModule, params: Dict, obfuscate: bool =
 
         script = moduleCode
 
-        # ridiculous escape format
-        groups = " ".join(['"\\""'+group.strip().strip("'\"")+'"""' for group in params["Groups"].split(",")])
-
         # build the custom command with whatever options we want
-        command = '""misc::addsid '+params["User"] + ' ' + groups
-
+        command = f'"sid::add /sam:{params["User"]} /new:{params["Group"]}"'
```
Comment (Cx01N) [inline empire/server/modules/powershell/persistence/misc/add_sid_history.py:32]: Change "Group" to "Groups" and this should be good to go.

**Review by Cx01N — CHANGES_REQUESTED** [review]

File: empire/server/modules/powershell/persistence/misc/add_sid_history.py (hunk 1 above)
Comment (Cx01N) [reply empire/server/modules/powershell/persistence/misc/add_sid_history.py:32]: Looks like there is another issue that comes back when making that change. Did you see this error?
image

Comment (ilanisme) [reply empire/server/modules/powershell/persistence/misc/add_sid_history.py:32]: Hey, Thanks for your time! 
Regarding groups i'm not sure but I think mimikatz supports only one group at a time, correct me if im wrong.
About the second error I think that happens in parsing mimikatz response, I'll add that change as well

File: empire/server/modules/powershell/persistence/misc/add_sid_history.yaml [hunk 2]
```diff
@@ -26,7 +26,11 @@ options:
     required: true
     value: ''
   - name: Groups
```
Comment (Cx01N) [inline empire/server/modules/powershell/persistence/misc/add_sid_history.yaml:28]: Thanks for that. That makes sense to me then, just update the yaml then so its 'Group' so the variable is called correct in the py file.

**Review by Cx01N — CHANGES_REQUESTED** [review]

File: empire/server/modules/powershell/persistence/misc/add_sid_history.yaml (hunk 2 above)
Comment (ilanisme) [reply empire/server/modules/powershell/persistence/misc/add_sid_history.yaml:28]: Got it, my bad. Thanks

**Review by Cx01N — APPROVED** [review]

**Pull request merged**